docroot/profiles/custom/cgov_site/modules/custom/cgov_syslog/cgov_syslog.module

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+
+/**
+ * Implementes hook_install().
+ */
+function cgov_syslog_install(): void {
+  _cgov_syslog_override_config();
+}
+
+/**
+ * Replace the entirety of syslog.settings with our own settings.
+ * @return void
+ */
+function _cgov_syslog_override_config(): void {
+  $override_config = \Drupal::config('cgov_syslog.settings')->getRawData();
+
+  if (!empty($override_config)) {
+    $syslog_config = \Drupal::configFactory()->getEditable('syslog.settings');
+    foreach ($override_config as $key => $value) {
+      $syslog_config->set($key, $value);
+    }
+    $syslog_config->save();
+  }
+}
+
+/**
+ * Update hook to replace syslog.settings with ours.
+ */
+function cgov_syslog_update_9001(): void {
```

Comment:
update hooks go into .install files.

Also why do we need the override config here? Why not set the syslog config from the .install for new sites? and set it in an update hook?

Reply:
The difficulty I ran into is that putting a copy of syslog.settings into the config/install results in a message because the settings have already been loaded by syslog and our settings (the new format string) doesn't get used.

It's annoying, but I didn't see a good alternative.